source/configure/deprecated-configuration-settings.rst

```diff
@@ -382,6 +381,42 @@ The port used for streaming data between servers.
 | This feature's ``config.json`` setting is ``"StreamingPort": ":8075"`` with string input. |
 +-------------------------------------------------------------------------------------------+
 
+Maximum idle connections for high availability
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+*Deprecated. Not used in Mattermost v7.0 and later*
+
++-----------------------------------------------------------------+------------------------------------------------------------------------+
+| The maximum number of idle connections held open from one       | - System Config path: N/A                                              |
+| server to all others in the cluster.                            | - ``config.json`` setting: ``".ClusterSettings.MaxIdleConns: 100,``    |
+|                                                                 | - Environment variable: ``MM_CLUSTERSETTINGS_MAXIDLECONNS``            |
+| Numerical input. Default is **100**.                            |                                                                        |
++-----------------------------------------------------------------+------------------------------------------------------------------------+
```

Comment:
The formatting is different to the other deprecated settings in the file. Is that intended?

Reply:
The only existing one I could find is `Enable publicly-shared boards` (https://github.com/mattermost/docs/blob/deprecate-legacy-configs/source/configure/deprecated-configuration-settings.rst#enable-publicly-shared-boards).

I'm 0/5 on what format to use. 1/5 that it looks a bit inconsistent to use different formats.